css-positioning/absolute.css

```diff
@@ -0,0 +1,46 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+
+  /* top: 20px;
+  left: 245px; */
+  top: 1rem;
+  right: -12rem;
+  width: 250px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 166px;
```

Comment:
This is a _**big**_ movement. Instead of moving this from the top, you should try starting from the `bottom`